metaci/build/models.py

```diff
@@ -283,6 +283,9 @@ def get_time_qa_start(self):
     def get_time_qa_end(self):
         return self.get_build_attr("time_qa_end")
 
+    def get_commit(self):
+        return f"{self.commit[:8]}..."
```

Comment:
Let's leave off the ellipsis. It'll make it easier to copy, and using an abbreviated git commit hash is a common convention.